### ISSUE TITLE
Keyboard focus indicator missing in item list

### ISSUE THREAD

**Issue by robertknight** [robertknight/passcards#23]
The change to add ripple animations to buttons and the item list in the web UI broke the keyboard focus indicator which indicates which item is selected in the item list when navigating it using the keyboard.
